lib/tests/server.ts

```diff
@@ -164,33 +160,35 @@ const kas: RequestListener = async (req, res) => {
         return;
       }
 
-      const rewrap = JSON.parse(requestBody as string) as RewrapBody;
+      const rewrap = fromJson(UnsignedRewrapRequestSchema, JSON.parse(requestBody as string));
       console.log('[INFO]: rewrap request body: ', rewrap);
       const clientPublicKey = await pemPublicToCrypto(rewrap.clientPublicKey);
       if (!clientPublicKey || clientPublicKey.type !== 'public') {
         res.writeHead(400);
         res.end('{"error": "Invalid client public key"}');
         return;
       }
-      const isZTDF = !rewrap.keyAccess.header;
+      const kaoheader = rewrap.requests?.[0]?.keyAccessObjects?.[0]?.keyAccessObject?.header;
```

Comment:
_nit, non-blocker_:

The object at `rewrap.requests?.[0]?.keyAccessObjects?.[0]?.keyAccessObject` is accessed a few times here, assigning it to a new const would improve readability 
```
const keyAccessObject = rewrap.requests?.[0]?.keyAccessObjects?.[0]?.keyAccessObject```